src/Behat/Testwork/Output/Exception/FormatterNotFoundException.php

```diff
@@ -33,6 +33,7 @@ class FormatterNotFoundException extends InvalidArgumentException implements Out
     public function __construct($message, $name)
     {
         parent::__construct($message);
+        $this->name = $name;
```

Comment:
This constructor parameter was not used but it actually needs to be used to set our private property

Reply:
Is it worth just making that a promoted property at this point?